data/en/echo.json

```diff
@@ -11,6 +11,19 @@
 	"engines": {
 		"lucee": {"minimum_version": "", "notes": "", "docs": "https://docs.lucee.org/reference/functions/echo.html"}
 	},
-	"examples": [],
+	"examples": [
+		{
+			"title": "Simple Example",
+			"description": "Prints a simple string.",
+			"code": "echo(\"hello world\");",
+			"result": "Prints the string 'hello world'"
```

Comment:
@CraicOverflow89 the result should be the actual result, of the code, so in this case it should just be `"hello world"` instead of `"Prints the string 'hello world'"`

The reason for this is that in some cases the result is actually tested automatically by our continuous integration scripts.